app.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'sinatra'
+require 'sinatra/reloader'
+require 'erb'
+require 'sanitize'
+require_relative 'lib/memo'
+
+def read_memos
+  File.open('./memos.json') do |file|
+    JSON.parse(file.read, { symbolize_names: true }).map do |key, value|
+      Memo.new(key, value)
+    end
```

Comment:
このままでもよいですが、たとえば、 

```suggestion
    JSON.parse(file.read, { symbolize_names: true }).each_with_object({}) do |(id, memo_json), memos|
      memos[id] = Memo.new(id, memo_json)
    end
```
のようにするなど、 Hashとして返すようにしてあげるとメモのidでそのまま検索できて便利かなと思いました。

Reply:
Hashに格納するように変更しました！